controllers/tools/index.js

```diff
@@ -7,12 +7,18 @@ const { toolsSecurityHeaders } = require('../../middleware/securityHeaders');
 const auth = require('../../middleware/authed');
 const cached = require('../../middleware/cached');
 const feedbackService = require('../../services/feedback');
+const applicationService = require('../../services/applications');
 const orderService = require('../../services/orders');
 const surveysService = require('../../services/surveys');
 
 const pagelistRouter = require('./pagelist');
 const seedRouter = require('./seed');
 
+const availableApplicationModels = [
+    require('../apply/building-connections/form-model'),
+    require('../apply/reaching-communities/form-model')
+];
+
```

Comment:
I feel sliiiiiiiightly weird about this, but I couldn't think of a better way to retrieve the details of the form and validate that it really exists. Suggestions welcome!

Reply:
Hmm yeah, what do we really need from the form model? Looks like just the title. Could we store that as an additional field so we're only querying against the applications table and don't need to worry too much about the original form model?